content/security/docs/incidents.md

```diff
@@ -33,6 +33,15 @@ kubectl get pods -o json --namespace <namespace> | \
     "\(.metadata.name) \(.spec.nodeName)"'
 ```
 
+### Identify the offending Pods and worker nodes using container image
+In some cases, you may discover that a container image being used in pods on your cluster is malicious or compromised. You should consider all the pods using the container image compromised. You can identify all the pods using the image and nodes they are running on with the following command:
+```
+kubectl get pods -o json --all-namespaces | \
+    jq -r '.items[] | 
+		select(.spec.containers[] | .image == "rastasheep/ubuntu-sshd:14.04") | 
```

Comment:
You probably want to use an environment variable for the name of the image otherwise you will only see results for pods with the `ratasheep/ubuntu-sshd:14.04` image. Folks of all skill levels use these guides. We shouldn't assume they know to replace the image name.

Reply:
Will fix this